src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -366,17 +368,16 @@ private void updateCaseDeadlines(CaseData caseData, UUID stageUUID, LocalDate da
         auditClient.updateCaseAudit(caseData, stageUUID);
     }
 
-    private void updateStageDeadlines(CaseData caseData) {
+    public void updateStageDeadlines(CaseData caseData) {
```

Comment:
This was copied and pasted and used in multiple places and is still used in 4 places.. but outside of the scope of this PR.